content/working_with/integration/jenkins-plugin.md

```diff
@@ -5,14 +5,14 @@ weight = 24
 alwaysopen = false
 +++
 
-The Jenkins Plugin for Cloudify, provided with version 5.0.5 onward, allows Jenkins authors to embed Cloudify functions within jobs and pipelines, thus simplifying the usage of the Cloudify Manager through Jenkins.
+The Jenkins Plugin for {{< param product_name >}}, provided with version 5.0.5 onward, allows Jenkins authors to embed {{< param product_name >}} functions within jobs and pipelines, thus simplifying the usage of the {{< param cfy_manager_name >}} through Jenkins.
```

Comment:
I'm wondering do we want to also add parameters for versions? (which can differ for customers)

Reply:
This may be phase 2.